.ado/windows-vs-pr.yml

```diff
@@ -328,6 +242,89 @@ stages:
                 React.Windows.Desktop.DLL\**
                 React.Windows.Desktop.Test.DLL\**
 
+      - job: CliInit
+        displayName: Verify react-native init
+        pool:
+          vmImage: $(VmImage)
+        timeoutInMinutes: 30 # how long to run the job before automatically cancelling
+        cancelTimeoutInMinutes: 5 # how much time to give 'run always even if cancelled tasks' before killing them
+        steps:
+          - checkout: self # self represents the repo where the initial Pipelines YAML file was found
+            clean: true # whether to fetch clean each time
+            # fetchDepth: 2 # the depth of commits to ask Git to fetch
+            lfs: false # whether to download Git-LFS files
+            submodules: false # set to 'true' for a single level of submodules or 'recursive' to get submodules of submodules
+            persistCredentials: false # set to 'true' to leave the OAuth token in the Git config after the initial fetch
+
+          # First do a build of the local package, since we point the cli at the local files, it needs to be pre-built
+          - task: CmdLine@2
+            displayName: yarn install (local react-native-windows)
+            inputs:
+              script: yarn install --frozen-lockfile
+
+          - task: CmdLine@2
+            displayName: yarn build (local react-native-windows)
+            inputs:
+              script: yarn build
+              workingDirectory: vnext
+
+          # yarn ends up copying the whole node_modules folder when doing an install of a file package
+          # Delete node_modules, so that resolution is more like when installing from a published npm package
+          - task: CmdLine@2
+            displayName: Remove node_modules
+            inputs:
+              script: rd /S /Q node_modules
+              workingDirectory: vnext
+
+          - task: CmdLine@2
+            displayName: Install react-native cli
+            inputs:
+              script: npm install -g react-native-cli
+
+          - task: CmdLine@2
+            displayName: Init new project
+            inputs:
+              script: react-native init testcli --version 0.60.6
```

Comment:
This needs to be the old version

Reply:
Noted. I asked that PR author to parameterize this value.

Clearly, I was ignored.